test/integration/controlplane/transformation/kms_transformation_test.go

```diff
@@ -625,6 +625,8 @@ resources:
 		defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, "AllBeta", true)()
 		// Need to enable this explicitly as the feature is deprecated
 		defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.KMSv1, true)()
+		// Need to disable this explicitly as this feature affects rest apis.
+		defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.EmulationVersion, false)()
```

Comment:
Eventually we need to remove the feature gate entirely.. what do we do then?  I think I'd rather not have this "SetFeatureGate" call here so that when we switch this feature gate to on-by-default we're forced to adapt..